packages/core/components/AnnotationFilterForm/index.tsx

```diff
@@ -33,6 +33,7 @@ export default function AnnotationFilterForm(props: AnnotationFilterFormProps) {
     const dispatch = useDispatch();
     const allFilters = useSelector(selection.selectors.getFileFilters);
     const fuzzyFilters = useSelector(selection.selectors.getFuzzyFilters);
+    const isQueryingAicsFms = useSelector(selection.selectors.isQueryingAicsFms);
```

Comment:
Nit: is it possible for some of the alternative datasources to support fuzzy, while others do not?  in which case the isQueryingAicsFms would be better named 'isQueryingSourceWithFuzzy'  or something similar but less awkward?

Reply:
Fair point! In the current setup, with all non-FES sources we do fuzzy matching by default on our end, but makes sense to prepare for the possibility